src/app/components/layouts/column-container/terra-two-columns-container.component.scss

```diff
@@ -0,0 +1,7 @@
+@media (max-width: 767px)
```

Comment:
isn't there a scss variable that can be used for the breakpoint?

Reply:
not in components by the look of it